src/administration/web/environments.md

```diff
@@ -1,6 +1,6 @@
 # Platform.sh Environments
 
-[Platform.sh](https://platform.sh) helps a coder with the development workflow by making it easy to manage multiple environments, including the Master environment which runs the production website. It's precisely like a "development" or a "staging" server.. except they are created on the fly, and they are absolutely identical copies of their parent environments.
+[Platform.sh](https://platform.sh) helps a coder with the development workflow by making it easy to manage multiple environments, including the Master environment which runs the production website. It's precisely like a "development" or a "staging" server... except they are created on the fly, and they are absolutely identical copies of their parent environments.
```

Comment:
An ellipsis is 3 periods.  Two doesn't make sense.  We could potentially swap it out for an ellipsis entity? :smile:

Reply:
I'm good either way.